api/src/pcapi/core/users/api.py

```diff
@@ -1580,6 +1583,111 @@ def anonymize_beneficiary_users(*, force: bool = False) -> None:
     db.session.commit()
 
 
+def anonymize_pro_users() -> None:
+    """
+    Anonymize pro account which have not connected for at least 3 years and are either:
```

Comment:
```suggestion
    Anonymize pro accounts which have not connected for at least 3 years and are either:
```
Ou `pro_users`, comme le nom de la fonction :p